content/en/blog/2023-03-01-february-update/index.md

```diff
@@ -0,0 +1,259 @@
+  Chainguard team and Wolfi maintainers are shipping updates for the Flux
+  package on a regular basis.
+
+Features and improvements
+
+- Recreate immutable resources (e.g. Kubernetes Jobs) by annotating
+  or labeling them with `kustomize.toolkit.fluxcd.io/force: enabled`.
+- Support for HTTPS bearer token authentication for Git repositories.
+- Improve memory usage by disabling the caching of `Secret` and
+  `ConfigMap` resources in all controllers.
+- Better observability with progressive status updates for Sources
+  (Git, OCI, Helm, S3 Buckets).
+- Allow extracting the OCI artifact SHA256 digest for Cosign with
+  `flux push artifact -o json`.
+- Track CRDs managed by Flux, `flux trace` and `flux tree` will show
+  which HelmRelease deployed which CRDs.
+- Allow the Flux GitHub Action to use a GitHub token when checking
+  for updates to avoid rate limiting.
+
+Documentation
+
+- Security: [Software Bill of Materials](/flux/security/#software-bill-of-materials)
+- Security: [SLSA Provenance Attestations](/flux/security/#slsa-provenance-attestations)
+- Security: [Scanning Flux images for CVEs](/flux/security/#scanning-for-cves)
+
+Big thanks to all the Flux contributors that helped us with this release!
+
+### Security news
+
+### Flagger x.y.z
+
+### Flux Ecosystem
+
+<!--
+
+If you add entries to this subsection, please don't use "we" as it gets
+confusing from which perspective the whole blog post is written and who
+"we" is. The whole post is meant to be from the perspective of the Flux
+community. Better to write:]
+
+- "Since the new release of X, it supports Y." or
+- "Team lead X says: 'we have put a lot of effort into Y and are
+   really proud of the performance results' ..." or
+- "The team has been working on ..."
+
+-->
+
+#### Flux Subsystem for Argo
+
+#### Terraform-controller
+
+#### Weave GitOps
+
+- <https://github.com/weaveworks/weave-gitops/releases/tag/v0.17.0>
+- <https://github.com/weaveworks/weave-gitops/releases/tag/v0.16.0>
+
+#### Azure GitOps
+
+#### VS Code GitOps Extension
+
+#### New additions to the Flux Ecosystem
+
+## Recent & Upcoming Events
+
+It's important to keep you up to date with new features and developments
+in Flux and provide simple ways to see our work in action and chat with
+our engineers.
+
+### Recent Events (ICYMI) 📺
+
+We feel blessed to have such a big community of users, contributors and
+integrators and so many are happy to talk about their experiences. In
+February here are a couple of talks we would like to highlight.
+
+Here is a list of additional videos and topics we really enjoyed -
+please let us know if we missed anything of interest and we will make
+sure to mention it in the next post!
+
+### Upcoming Events 📆
+
+We are happy to announce that we have a number of events coming up in
+March - tune in to learn more about Flux and GitOps best practices,
+get to know the team and join our community.
+
+### Flux Bug Scrub
+
```

Comment:
@kingdonb Anything we should put in here apart from the updated dates?

Reply:
```suggestion
The Flux Bug Scrub has undertaken a slightly new format, where we'll be spending more time working on our own code, and even building some new software under the aegis of the "flux-community" organization! We'll still have our familiar spreadsheet every week, but we will be conversely spending less time per-meeting on curating the list of issues across the FluxCD org than we have been, which was the primary focus of these meetings before, and spending more time on code, and discussing cross-cutting issues that might be important for Flux releases (or other objectives that Flux maintainers have on their plates on a day-to-day basis.)

We are also evaluating the possibility of a new, more inclusive "Late Late" Bug Scrub, which would aim to be more accessible to people in the Asia/Pacific time zones, Australia, and even those in Eastern Europe who might not be able to make these earlier times. If you are in these time zones, it'll actually be "Early Work Hours" if we've calibrated this correctly. The Bug Scrub community is small but growing; we need your feedback to make this a hit (which you can provide asynchronously in the Flux channel on CNCF slack, please remember to @ KingdonB if you are interested!)

The addition of another meeting at a new time of day, is intended to provide a little extra time for the curation of issues to go along with this new format change, and also to balance the scales somewhat regionally (which currently, the timing favors Flux contributors or new joiners who are using Flux at their day jobs in both the Americas and Europe.)
```

Reply:
I gave the above a round of edits and I think it flows much better, more focused on the most interesting points. I'm going to go ahead and commit it, but feel free to edit further.